catalog/core/catalog-core-camelcomponent/src/main/java/ddf/camel/component/catalog/content/ContentProducerDataAccessObject.java

```diff
@@ -166,33 +180,34 @@ public void createContentItem(FileSystemPersistenceProvider fileIdMap, ContentEn
             if (createResponse != null) {
                 List<Metacard> createdMetacards = createResponse.getCreatedMetacards();
 
-                for (Metacard metacard : createdMetacards) {
-                    fileIdMap.store(key, metacard.getId());
-                    LOGGER.debug("content item created with id = {}", metacard.getId());
+                if (safeKey != null) {
+                    fileIdMap.store(safeKey,
+                            createdMetacards.get(0)
+                                    .getId());
                 }
+                LOGGER.debug("content item created with id = {}",
+                        createdMetacards.stream()
+                                .map(Metacard::getId));
```

Comment:
❗️ This logger statement is wrong. It will `toString()` the stream returned from the `map` method. To match above, you want to get `createdMetacards.get(0).getId()`. Conversely, 
```java
createMetacards.stream()
    .map(Metacard::getId)
    .findFirst()
    .orElse("XXX")
```